book.rb

```diff
@@ -0,0 +1,15 @@
+require_relative('./rental')
+class Book
+  attr_accessor :title, :author
+
```

Comment:
- [ ] Please make sure the `rentals` variable is readable and writable in the Book class to work as expected. 
  - You can consider the below code.

```ruby
attr_accessor :title, :author, :rentals